test/e2e/allocation.new.test.js

```diff
@@ -0,0 +1,52 @@
+import { Selector, t } from 'testcafe'
+
+import { pmuUser } from './_roles'
+import { allocations, newAllocation } from './_routes'
+import { allocationDetails, allocationCriteria, page } from './pages/'
+
+const selectors = {
```

Comment:
Can we use [Page Objects](https://devexpress.github.io/testcafe/documentation/guides/concepts/page-model.html) for defining selectors please. This keeps inline with the existing pattern and tries to keep selectors to one place so if they change we don't have to update them through test files.

Reply:
👍